e2e/src/api/specs/asset.e2e-spec.ts

```diff
@@ -1137,7 +1137,7 @@ describe('/asset', () => {
           type: AssetTypeEnum.Image,
           originalFileName: '14bit-uncompressed-(3_2).arw',
           resized: true,
-          fileCreatedAt: '2016-01-08T15:08:01.000Z',
+          fileCreatedAt: '2016-01-08T14:08:01.000Z',
```

Comment:
Why the timestamp change?

Reply:
https://github.com/photostructure/exiftool-vendored.js/issues/187